time timeStamp
bool frontCar
bool rearCar
float32 rearDistance